skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldCommandLineState.kt

```diff
@@ -60,6 +64,24 @@ class SkaffoldCommandLineState(
             throw ExecutionException(message("skaffold.no.file.selected.error"))
         }
 
+        //call skaffold version --- look at createProcess function, the command list lets you know if skaffold isn't on the system by process of elimination
+        if (Paths.get("skaffold").exists()) {
+            val testSkaffoldCommand = Paths.get("skaffold").toString() + " " + (executionMode.modeFlag)
+            try {
+                SkaffoldProcess(
+                    SkaffoldExecutorService.instance.createProcess(
+                        File(projectBaseDir.path),
+                        listOf(testSkaffoldCommand)
+                    ),
+                    commandLine = testSkaffoldCommand//skaffold executable path
+                )
+            }catch (e: Exception){
+                throw ExecutionException("skaffold path in system is incorrect")
```

Comment:
any time we have UI visible messages in the code, we should put them in the message bundle. see line 64 for an existing example of this:

`throw ExecutionException(message("skaffold.no.file.selected.error"))`